### DISCUSSION TITLE
Multiple accounts support

### DISCUSSION THREAD

**Discussion by hosembafer** [community/community#3827] [Enterprise] [answered]
It doesn’t make sense to sing out/in each time between personal and work accounts to view something, I'm not about enterprise server, but regular accounts

**Comment by nplasterer** [comment]
Hey thanks for your feedback! Feel free to upvote on this [topic related to multiple account support](https://github.com/github/feedback/discussions/3305) to help boost awareness of this request! - #3305

**Comment by eliperkins** [comment, marked as answer]
Hey @hosembafer , we've shipped functionality this week to allow you to log in within your personal and work accounts at a the same time! [Read more about it in our Changelog post](https://github.blog/changelog/2023-08-15-log-in-with-multiple-github-accounts-on-github-mobile/).

<img width="1200" alt="image" src="https://github.com/community/community/assets/1051453/d2ba902e-2199-4625-948a-532d64a6f44e">

Download or update GitHub Mobile today from the [Apple App Store](https://apps.apple.com/us/app/github/id1477376905) or [Google Play Store](https://play.google.com/store/apps/details?id=com.github.android) to get started.
